### ISSUE TITLE
Accessibility: "Back" button missing a Talkback-friendly label in certain areas of the settings section

### ISSUE THREAD

**Issue by Murcielago7384** [signalapp/Signal-Android#14924]
### Guidelines

- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] This isn't a feature request or a discussion topic

### Bug description

When using the Talkback screen reader in the Signal Android app, the 'back' button is missing a spoken Talkback label one level into the settings section of the app.

To reproduce:
1. Make sure Talkback is running.
2. Open Signal > settings.
3. Click on any category, but for example's sake, "chats".
4. Press the 'back' button in the upper left-hand corner of the screen. Notice that Talkback simply announces "button".

I am not sure if any other areas of the app are missing a similar label, but this is the one that I came across after recently switching to Graphene OS.

### Device

Google Pixel 8A

### Android version

Android 17 (Graphene)

### Signal version

8.21.5
